edx-platform/pearson-vue-theme/lms/static/sass/partials/lms/theme/_extras.scss

```diff
@@ -332,6 +298,118 @@ div.toggle-form {
   background-size: cover;
 }
 
+.homepage-content {
+  min-height: calc(100vh - 60px);
+  display: grid;
+  grid-template-columns: 40% 60%;
+  margin-top: -20px;
+
+  &::before {
+    content: '';
+    position: absolute;
+    top: 70px;
+    left: 0;
+    height: 70%;
+    width: 100%;
+    background: url('/static/pearson-vue-theme/images/CertPREP_homepage_background.png');
```

Comment:
```suggestion
    background: url('#{$static-path}/images/CertPREP_homepage_background.png');
```

Reply:
Thanks, I didn't know this could be used